controller.js

```
@@ -51,3 +52,19 @@ exports.getcommentsById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postnewcomment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+  if (isNaN(Number(article_id))) {
+    return res.status(400).send({ msg: "Bad Request!" });
+  }
+  if (!username || !body) {
```

If one of the required fields is missing then PSQL should throw an error for you because when creating the tables these have been marked as not null